packages/forms/src/fields/CustomObjectField.js

```diff
@@ -0,0 +1,151 @@
+import React, { PropTypes } from 'react';
+import { Icon } from 'react-talend-components';
+import { orderProperties, retrieveSchema } from 'react-jsonschema-form/lib/utils';
+
+function createCustomObjectField(title) {
+	class CustomObjectField extends React.Component {
```

Comment:
I don't like this name :)
Something more descriptive would be better, CollapsibleFieldset ?